tools/packaging/postinst.sh

```diff
@@ -20,7 +20,7 @@ set -e
 umask 022
 
 if ! getent passwd istio-proxy >/dev/null; then
-    if  find /etc/*release | grep -Eiq 'centos|redhat'; then
+    if [ -x /usr/sbin/useradd ]; then
```

Comment:
is `command -v useradd` more robust? useradd could be in any folder?

Reply:
Both approaches are shell builtin which is ideal. While the -x looks for both the existence of the file and makes sure the file is executable. The command -v only displays the filename. Also, it might be a little cleaner to go with the -x in this context.
All the major distros such as Redhat, centos, and SUSE that leverage `useradd` and `groupadd` install these utilities in /usr/sbin/.

Reply:
IMO relying on the path is a bad pattern. https://stackoverflow.com/questions/592620/how-can-i-check-if-a-program-exists-from-a-bash-script recommends `command` (or even both in the 2nd answer)